src/cljs/notifikator/util.cljs

```diff
@@ -0,0 +1,18 @@
+(ns notifikator.util)
+
+(defn remove-by-id
+  "Looking for an element with the specified id and removing it
+   from the collection."
+  [coll id]
+  (remove #(= id (:id %)) coll))
+
+(defn next-msg-id
+  "Return 0 if there are no messages.
+   If not empty, find the max ID and return its incremente value."
+  [{:keys [messages]}]
+  (if (or (empty? messages)
+          (nil? messages))
+    0
+    (-> (apply max-key :id messages)
+        :id
+        inc)))
```

Comment:
minor: you could also consider using a [random uuid](https://github.com/lbradstreet/cljs-uuid-utils)

Reply:
Actually a good idea. Especially considering I can remove messages in any order. Can also drop the next-id calculation logic. But I'll keep my simple IDs for now. Two reasons:
1. Don't want an extra dependency in this small POC task.
2. Smaller and easier to read in HTML/React dev tools.

But yeah, for actual prod I would probably go with random UUIDs to exclude the complexity of tracking the next ID and thus potential bugs.